src/calendar/date/CalendarUtils.ts

```diff
@@ -309,11 +309,18 @@ export function colorForBg(color: string): string {
 	return isColorLight(color) ? "black" : "white"
 }
 
+
+/**
+ * Function which sorts events into the "columns" and "rows" and renders them using {@param renderer}.
+ * Columns are abstract and can be actually the rows. A single column progresses in time while multiple columns can happen in parallel.
+ * @param eventsTakeUpWholeColumn {Function} when false, we can put multiple events one after another in a single column, otherwise only one event can be
+ * in one column on a single day (it will "stretch" events from the day start until the next day).
+ */
 export function layOutEvents(
 	events: Array<CalendarEvent>,
 	zone: string,
 	renderer: (columns: Array<Array<CalendarEvent>>) => ChildArray,
-	handleAsAllDay: boolean,
+	eventsTakeUpWholeColumn: boolean,
```

Comment:
should probably be an enum, call sites are unclear. Maybe CollisionLayout.Stack and CollisionLayout.Squash or w/e